W-18065800-pce-studio-certificate-update-kt (#605)
mulesoft/docs-studio: modules/ROOT/pages/_partials/pce-configuration-macos.adoc

--
openssl s_client -connect your.dsm.name.com:443 -showcerts > my-certificate.pem
--
+
[NOTE]

> Starting with Anypoint Studio 7.17, the default JDK version for Studio is Java 17. To ensure proper functionality, add the certificate to the default JDK version that Anypoint Studio supports. +

Reviewer: ```suggestion
Starting with Studio 7.17, the default JDK version for Studio is Java 17. To ensure proper functionality, add the certificate to the default JDK version that Studio supports. +
```

Reviewer: Globally

> * If you use an external JDK, add the certificate to the external JDK that you use with Anypoint Studio. +

Reviewer: ```suggestion
* If you use an external JDK, add the certificate to the external JDK that you use with Studio. +
```

Reviewer: Globally

> * If you use the default JDK provided by Anypoint Studio, add the certificate to the default JDK, regardless of the JDK version you use in your project.

Reviewer: ```suggestion
* If you use the default JDK Studio provides, add the certificate to the default JDK, regardless of the JDK version you use in your project.
```

Reviewer: Globally
